### PR TITLE
Native/2.10.5

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

File: common/config/rush/pnpm-lock.yaml [hunk 1]
```diff
@@ -305,7 +305,7 @@ dependencies:
   react-redux: 7.2.2_bbabd8c34ea235719dc50e75b43f85a4
   react-resize-detector: 4.2.3_react-dom@16.14.0+react@16.14.0
   react-router-dom: 5.2.0_react@16.14.0
-  react-select: 3.1.0_react-dom@16.14.0+react@16.14.0
+  react-select: 3.1.1_react-dom@16.14.0+react@16.14.0
```
Comment (pmconne) [inline common/config/rush/pnpm-lock.yaml:308]: @bsteinbk @NancyMcCallB ui-core is failing with the following errors. Possibly related to this package? Any suggestions?
```
--[ FAILURE: @bentley/ui-core ]--------------------[ 2 minutes 48.0 seconds ]--

src/ui-core/select/ThemedSelect.tsx(28,16): error TS2314: Generic type 'ValueType' requires 2 type argument(s).
src/ui-core/select/ThemedSelect.tsx(85,16): error TS2314: Generic type 'SelectComponentsConfig' requires 2 type argument(s).
src/ui-core/select/ThemedSelect.tsx(91,18): error TS2314: Generic type 'ValueType' requires 2 type argument(s).
src/ui-core/select/ThemedSelect.tsx(158,22): error TS2314: Generic type 'ValueType' requires 2 type argument(s).
src/ui-core/select/ThemedSelect.tsx(190,11): error TS2314: Generic type 'ValueType' requires 2 type argument(s).
src/ui-core/select/ThemedSelect.tsx(193,28): error TS2314: Generic type 'MenuProps' requires 2 type argument(s).
```

Comment (pmconne) [reply common/config/rush/pnpm-lock.yaml:308]: Looks like changes to the type definitions broke this 7 days ago. Do I lock you to previous version or update your code? If the latter, what "is multi" type do you want? https://github.com/DefinitelyTyped/DefinitelyTyped/commit/e52c6aae2ec85e8ff96f179db412a861317815d2

Comment (pmconne) [reply common/config/rush/pnpm-lock.yaml:308]: @bsteinbk recommended specifying `false` for the new `IsMulti` type parameter. New errors. Sorry folks, I am lost with this React stuff. Please advise.
```
src/test/select/ThemedSelect.test.tsx:106:70 - error TS2739: Type '{ label: string; value: string; }[]' is missing the following properties from type 'OptionType': value, label

106     const { container } = render(<ThemedSelect options={cityChoices} value={[cityChoices[3], cityChoices[5]]} />);
                                                                         ~~~~~

  src/ui-core/select/ThemedSelect.tsx:190:3
    190   value?: ValueType<OptionType, false>;
          ~~~~~
    The expected type comes from property 'value' which is declared here on type 'IntrinsicAttributes & ThemedSelectProps'

src/test/select/ThemedSelect.test.tsx:110:85 - error TS2322: Type '{ label: string; value: string; }[]' is not assignable to type 'OptionType'.

110     const { container } = render(<ThemedSelect options={cityChoices} isMulti={true} value={[cityChoices[3], cityChoices[5]]} />);
                                                                                        ~~~~~

  src/ui-core/select/ThemedSelect.tsx:190:3
    190   value?: ValueType<OptionType, false>;
          ~~~~~
    The expected type comes from property 'value' which is declared here on type 'IntrinsicAttributes & ThemedSelectProps'
```

Comment (bsteinbk) [reply common/config/rush/pnpm-lock.yaml:308]: @NancyMcCallB, Paul is going to set isMulti to false based on my feedback.  So let me know if we need to change that at some point.

Comment (pmconne) [reply common/config/rush/pnpm-lock.yaml:308]: See prev reply, produces new errors.

Comment (NancyMcCallB) [reply common/config/rush/pnpm-lock.yaml:308]: I think we should lock to 3.1.0 for now and I'll filed a bug to look into it. Do you have any problems with that, @pmconne?

Comment (pmconne) [reply common/config/rush/pnpm-lock.yaml:308]: Nancy, the react-select package is not the problem, it's @types/react-select. AFAICT. See previously-linked DefinitelyType PR.

Comment (NancyMcCallB) [reply common/config/rush/pnpm-lock.yaml]: I locked the react-select dependencies and I'll look at the types in the morning. Let me know if there are still problems.

Comment (pmconne) [reply common/config/rush/pnpm-lock.yaml]: Thanks @NancyMcCallB.

**Review by kabentley — APPROVED** [review]

**Review by grigasp — APPROVED** [review]

**Review by GerardasB — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**